### PR TITLE
Add `schema_dump` docs to the multiple databases docs [skip ci]

### PR DESCRIPTION
**Pull request by fatkodima**
Follow up to https://github.com/rails/rails/pull/42804 and https://github.com/rails/rails/pull/43530.

cc @eileencodes

### CODE REVIEW

File: guides/source/active_record_multiple_databases.md [hunk 1]
```diff
@@ -443,7 +446,7 @@ class ShardRecord < ApplicationRecord
 end
 ```
 
-If you're using shards, make sure to set the `migrations_paths` to the same path for
+If you're using shards, make sure to set the `migrations_paths` and `schema_dump` to the same paths for
```
[the remark this answers is not in the document]
Comment (zzak) [reply guides/source/active_record_multiple_databases.md:449]: Maybe:

> If you're using shards, make sure both `migrations_paths` and `schema_dump` remain unchanged for all the shards.

[the remark this answers is not in the document]
Comment (fatkodima) [reply guides/source/active_record_multiple_databases.md]: Thanks! I read it the same, but was looking for a better wording 😄

File: guides/source/active_record_multiple_databases.md [hunk 2]
```diff
@@ -102,6 +102,9 @@ Lastly, for new writer databases, you need to set the `migrations_paths` to the
 where you will store migrations for that database. We'll look more at `migrations_paths`
 later on in this guide.
 
+You can also configure the schema dump file by setting `schema_dump` to a custom schema file name
+or completely skip the schema dumping by setting `schema_dump: false`.
```
Comment (zzak) [inline guides/source/active_record_multiple_databases.md:106]: Maybe we could add a link to https://edgeguides.rubyonrails.org/active_record_migrations.html#schema-dumping-and-you

Comment (fatkodima) [reply guides/source/active_record_multiple_databases.md:106]: I honestly do not think that this is needed. It is expected, that people know what it is 🤔 .

Comment (zzak) [reply guides/source/active_record_multiple_databases.md:106]: I thought so too, probably safe to assume if they are setting up multiple dbs they already are familiar with that kind of stuff. Just thought I'd ask anyway :pray: Thanks for working on this!

**Review by zzak — APPROVED** [review]
Thank you!

**Review by eileencodes — APPROVED** [review]

**Pull request merged**